src/main/java/com/car/app/service/CarService.java

```diff
@@ -1,31 +1,21 @@
 package com.car.app.service;
```

Comment:
Please move the class the package control

Reply:
Please follow the package structure according to the standard: ECB

http://www.cs.sjsu.edu/~pearce/modules/patterns/enterprise/ecb/ecb.htm